src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -3,5 +3,5 @@
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
-    User register(User user);
+    User register(User user) throws UnacceptableUserExeption;
```

Comment:
`RuntimeException` is unchecked exception
```suggestion
    User register(User user);
```